### ISSUE TITLE
istanbul ignore else

### ISSUE THREAD

**Issue by ph-fritsche** [kentcdodds/kcd-scripts#218]
- `kcd-scripts` version: 11.2.0
- `node` version: any
- `npm` (or `yarn`) version: any

Relevant code or config

https://github.com/testing-library/user-event/pull/706/files#diff-79de2fcdb58def5d2000c20beddc4bb5a30488589e20c207181967dd0e75aa72R28

What you did:

Run CI with `"kcd-scripts": "^11.1.0"`.

What happened:

After release of `v11.2.0` CI fails due to uncovered lines.
Turns out istanbul expects the `/* istanbul ignore else */` right in between `else if` now.

The pre-commit hook reverts the change and therefore commits fixing this have to be committed with `--no-verify`. 

Reproduction repository:

https://github.com/testing-library/user-event/pull/706

Problem description:

`kcd-scripts lint` does not flag `/* istanbul ignore else */ else if`.
`kcd-scripts/husky` adds a pre-commit hook that changes the (now) correct `else /* istanbul ignore else */ if` back to `/* istanbul ignore else */ else if`.

Suggested solution:

Report a lint error for `/* istanbul ignore else */ else if`.
Lint-fix to `else /* istanbul ignore else */ if`.

**Comment by kentcdodds** [comment]
I'm not sure I understand what's going on. Is this a linting issue or a prettier issue? Either way I'm not sure what we could do in this project to fix that.

**Comment by ph-fritsche** [comment]
It's both
The linter does not report an error for `/* istanbul ignore else */ else if` although Istanbul does not honor this annotation.
The prettier does change the correct `else /* istanbul ignore else */ if` to `/* istanbul ignore else */ else if`.

If we can't fix this in this repo by adjusting some config, then maybe we can relay this problem to the dependency that could fix this.
Otherwise it will serve as a reference for the next poor soul that updates to `v11.2` and suddenly has a coverage quota violation without touching code or tests. ;)

**Comment by kentcdodds** [comment]
Yeah, I have no idea what caused the change in behavior and I'm afraid I don't have the bandwidth to investigate it :-/

**Comment by eps1lon** [comment]
We had the same problem in dom-testing-library. I just converted the `else if` to an `else { if }`: https://github.com/testing-library/dom-testing-library/pull/1013
